package/Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM registry.suse.com/bci/bci-base:15.3
+FROM ghcr.io/oracle/oraclelinux:7-slim
 
 ARG KUBERNETES_RELEASE=v1.21.12
 WORKDIR /usr/local/bin
-RUN set -x \
- && zypper -n install curl \
- && curl -fsSLO https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/amd64/kubectl \
- && chmod +x kubectl
+
+RUN yum-config-manager --enable ol7_optional_latest && \
+    yum-config-manager --enable ol7_addons
+
+RUN yum update -y && \
+    yum install -y bash curl && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne15/x86_64/ && \
+    yum install -y kubectl-1.23.11-1.el7.x86_64 && \
+    yum clean all
```

Comment:
You should do this to remove unneeded yum files
&& rm -rf /var/cache/yum

Reply:
fixed